docker-x64/Dockerfile

```
@@ -54,10 +54,10 @@ ENV PATH="/go/bin:/usr/local/go/bin:${PATH}"
 
 # Install vault: https://github.com/hashicorp/vault/blob/main/CHANGELOG.md https://releases.hashicorp.com/vault
 RUN curl -LO https://releases.hashicorp.com/vault/$VAULT_VERSION/$VAULT_FILENAME \
-    && echo "${VAULT_CHECKSUM}  ${VAULT_FILENAME}" | sha256sum --check \
-    && unzip $VAULT_FILENAME \
+    && echo "${VAULT_CHECKSUM} ${VAULT_FILENAME}" | sha256sum --check \
+    && unzip $VAULT_FILENAME -d /tmp/vault \
     && rm $VAULT_FILENAME \
-    && mv vault /usr/bin/vault \
+    && mv /tmp/vault/vault /usr/bin/vault \
     && chmod +x /usr/bin/vault
```

As above, we can tell `unzip` to put directly the `vault` file at the right location without inflating `LICENSE` in a single line:

```suggestion
    && unzip -o $VAULT_FILENAME -d /usr/bin -o vault \
    && rm $VAULT_FILENAME
```